test/data/conformance/only-vet.json

```diff
@@ -0,0 +1,75 @@
+{
+  "veteranFullName": {
+    "first": "FirstName",
+    "middle": "MiddleName",
+    "last": "ZZTEST",
+    "suffix": "Jr."
+  },
+  "mothersMaidenName": "Maiden",
+  "veteranSocialSecurityNumber": "111-11-1234",
+  "gender": "F",
+  "cityOfBirth": "Springfield",
+  "stateOfBirth": "AK",
+  "veteranDateOfBirth": "1923-01-02",
+  "maritalStatus": "Never Married",
+  "isVaServiceConnected": false,
+  "compensableVaServiceConnected": false,
+  "receivesVaPension": true,
+  "isEssentialAcaCoverage": true,
+  "facilityState": "CT",
+  "vaMedicalFacility": "689A4",
+  "wantsInitialVaContact": true,
+  "isSpanishHispanicLatino": true,
+  "isAmericanIndianOrAlaskanNative": true,
+  "isBlackOrAfricanAmerican": true,
+  "isNativeHawaiianOrOtherPacificIslander": true,
+  "isAsian": true,
+  "isWhite": true,
+  "veteranAddress": {
+    "street": "123 NW 5th St",
+    "city": "Washington",
+    "country": "USA",
+    "state": "DC",
+    "zipcode": "20005"
+  },
+  "email": "foo@example.com",
+  "homePhone": "1231241234",
+  "mobilePhone": "1235551234",
+  "understandsFinancialDisclosure": true,
+  "dateOfMarriage": "1983-05-10",
+  "sameAddress": true,
+  "cohabitedLastYear": true,
+  "provideSupportLastYear": false,
```

Comment:
Even though they don't impact the output (which is conditional on other items), I really should remove these and clean these samples up. Will do that now